clrfeatures.cmake

```diff
@@ -25,3 +25,9 @@ endif(NOT DEFINED FEATURE_DBGIPC)
 if(NOT DEFINED FEATURE_INTERPRETER)
   set(FEATURE_INTERPRETER 0)
 endif(NOT DEFINED FEATURE_INTERPRETER)
+
+if(NOT DEFINED FEATURE_NI_BIND_FALLBACK)
```

Comment:
One more suggestion - can you make this entire check under "if not Win32" as that will allow you to remove FEATURE_PAL from https://github.com/dotnet/coreclr/pull/11485/files#diff-1708de55e5220e56e33f7ddb0f465f9cR692?